ability_item_usage_furion.lua

```diff
@@ -49,12 +49,12 @@ local AbilityToLevelUp = {
 
 local TalentTree = {
```

Comment:
Why are there 8 talent options here? 
I see, it should be the selection order after level 25. I don't know if other heroes will choose talents after level 25.

Reply:
The furion code is written in this way. I really don't know why. 

It was playtested that furion and other heroes will choose talents after level 25 properly.

Reply:
That's interesting.